lib/Makefile

```diff
@@ -45,6 +45,8 @@ ZDICT_FILES := $(sort $(wildcard dictBuilder/*.c))
 ZDEPR_FILES := $(sort $(wildcard deprecated/*.c))
 ZSTD_FILES := $(ZSTDCOMMON_FILES)
 
+decompress/zstd_decompress_block.o : CFLAGS+=-fno-tree-vectorize
```

Comment:
I'm wondering if all compilers that use the Makefile support or ignore this flag. For example, people will compile zstd with `icc`, and also some ancient compilers.

We may have to test to see if the compiler supports this flag.

Reply:
Good point, they don't.